cognite_toolkit/cdf.py

```diff
@@ -341,7 +341,7 @@ def deploy(
         # Last, we need to get all the scoped access, as the resources should now have been created.
         print("[bold]EVALUATING auth resources scoped to resources...[/]")
         result = deploy_or_clean_resources(
-            AuthLoader.create_loader(ToolGlobals, target_scopes="resource_scoped_only"),
+            AuthLoader.create_loader(ToolGlobals, target_scopes="all"),
```

Comment:
This is the bugfix. The issue is that we are creating new Groups and not updating existing group capabilities. Since we had `resource_scoped_only`, we are filtering out the all scoped capabilities and then deleting and recreating the groups with only resource scoped capabilities. Meaning the all scoped capabilities created on line 319 are lost, and replaced with groups which has only the resource scoped capabilities.

Reply:
I think you need to update comments as well.
I believe the correct behaviour is to do as I initially implemented? I.e. the diff mechanism to ensure that we are not deleting when not necessary. This mechanism seems to crude. Without having mapped out what the various target_scopes actually do in the various scenarioes (clean, deploy) and for different existing states, I must admit I'm unable to immediate see that changes this to "all" will not have unintended effects.